<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationShow</class>
 <widget class="QDialog" name="InformationShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>14</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_information_eid">
      <property name="text">
       <string>Eid</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_information_eid"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_information_name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_information_name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_information_name_3">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_information_surname"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_information_name_2">
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_information_age"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_information_logout">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LogOut</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
